lighthouse-core/test/audits/load-fast-enough-for-pwa-test.js

```diff
@@ -75,8 +75,7 @@ describe('PWA: load-fast-enough-for-pwa audit', () => {
 
     const settings = {throttlingMethod: 'provided', throttling: {rttMs: 40, throughput: 100000}};
     const result = await FastPWAAudit.audit(artifacts, {settings, computedCache: new Map()});
-    expect(result.rawValue).toBeGreaterThan(2000); // If not overridden this would be 1582
-    expect(Math.round(result.rawValue)).toMatchSnapshot();
```

Comment:
wasn't sure why this was this way, but just a single number, so removed

Reply:
@brendankenny it's a lantern computed value it does it this way so that lantern changes can be updated with `-u` in jest

Reply:
could you revert with a comment?